lib/Migration/ResetSessionsBeforeYjs.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace OCA\Text\Migration;
+
+use OCA\Text\Db\SessionMapper;
+use OCA\Text\Service\DocumentService;
+use OCP\IConfig;
+use OCP\Migration\IOutput;
+use OCP\Migration\IRepairStep;
+
+class ResetSessionsBeforeYjs implements IRepairStep {
+	private IConfig $config;
+	private SessionMapper $sessionMapper;
+	private DocumentService $documentService;
+
+	public function __construct(IConfig $config,
+								SessionMapper $sessionMapper,
+								DocumentService $documentService) {
+		$this->config = $config;
+		$this->sessionMapper = $sessionMapper;
+		$this->documentService = $documentService;
+	}
+
+	/**
+	 * @return string
+	 */
+	public function getName(): string {
+		return 'Force-reset all Text sessions before Yjs migration';
+	}
+
+	/**
+	 * @param IOutput $output
+	 *
+	 * @return void
+	 */
+	public function run(IOutput $output): void {
+		$appVersion = $this->config->getAppValue('text', 'installed_version');
+
+		if (!$appVersion || version_compare($appVersion, '3.7.1') !== -1) {
```

Comment:
Should we still do this once also for instances upgraded to 26 during beta phase? Then I'd to a increase to 3.7.2 in appinfo and use that instead

Reply:
Good point! Changed now.